CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ you need to change the following line in `config/environments/production.rb`:
 - **decidim-core**: Removed horizontal scroll from process navigation bar. [\#2495](https://github.com/decidim/decidim/pull/2495)
 - **decidim-core**: Fixes the documentation regarding gems, libraries, plugins and modules. We should always use module, and use decidim-module-<engine_name> nomenclature for repositories naming [\#2481](https://github.com/decidim/decidim/pull/2481).
 - **decidim-core**: Users have notifications enabled by default unless they explicitly uncheck it during registration. [\#2517](https://github.com/decidim/decidim/pull/2517)
+- **decidim-core**: Decidim now requires Ruby 2.5.0 [\#2550](https://github.com/decidim/decidim/pull/2550)
```

Comment:
In the end there'll be no real user facing changes in this PR, so we should drop the change log entry.